tests/structures/test_imagelist.py

```diff
@@ -44,7 +44,10 @@ def test_imagelist_scriptability(self):
         image_tensor = torch.randn((image_nums, 10, 20), dtype=torch.float32)
         image_shape = [(10, 20)] * image_nums
 
-        ImageList_script = torch.jit.script(ImageList)
+        qualified_name = torch._jit_internal._qualified_name(ImageList)
+        ImageList_script = torch.jit._state._get_script_class(qualified_name)
+        if not ImageList_script:
+            ImageList_script = torch.jit.script(ImageList)
```

Comment:
what's the reason for this change?

Reply:
It seems that exporting RPN to torchscript will register `ImageList` to JIT. Without these changes we will get:
`RuntimeError: class '__torch__.detectron2.structures.image_list.ImageList' already defined.`

Reply:
I just run the test through the ways described [here](https://github.com/facebookresearch/detectron2/blob/master/tests/README.md) and get this error.
How about add `torch.jit.script(ImageList)` after `export_torchscript_with_patch_instance` in `test_rpn.py` and run this test file alone, This will cause the same error in my development environment.

Reply:
Turns out that a non-module class object cannot be scripted twice. This will be a better way to write unittest:
```diff
-        ImageList_script = torch.jit.script(ImageList)
+        def f(image_tensor, image_shape: List[Tuple[int, int]]):
+            return ImageList(image_tensor, image_shape)
 
-        ret = ImageList(image_tensor, image_shape)
-        ret_script = ImageList_script(image_tensor, image_shape)
+        ret = f(image_tensor, image_shape)
+        ret_script = torch.jit.script(f)(image_tensor, image_shape)
+        print(ret_script)
```

Reply:
done